create-cardholders-and-cards/server/dotnet/Program.cs

```diff
@@ -1,142 +1,145 @@
-using System.Text.Json;
-using System.Text.Json.Serialization;
+using Microsoft.AspNetCore.StaticFiles.Infrastructure;
 using Microsoft.Extensions.FileProviders;
-
 using Stripe;
 using Stripe.Issuing;
+using CardCreateOptions = Stripe.Issuing.CardCreateOptions;

```

Comment:
Thanks a ton for your feedback here, @pakrym-stripe. I'm curious what the benefit is for these using statements where we're creating aliases even though we have already imported all of Stripe.Issuing.